<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mapp</class>
 <widget class="QMainWindow" name="mapp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>437</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>mapp</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLineEdit" name="addrLineEdit">
    <property name="geometry">
     <rect>
      <x>-5</x>
      <y>0</y>
      <width>580</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="goPushButton">
    <property name="geometry">
     <rect>
      <x>581</x>
      <y>-1</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Go</string>
    </property>
   </widget>
   <widget class="QPushButton" name="mapBtn">
    <property name="geometry">
     <rect>
      <x>662</x>
      <y>-1</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Map</string>
    </property>
   </widget>
   <widget class="WebAxWidget" name="webBrowser">
    <property name="control" stdset="0">
     <string>{8856f961-340a-11d0-a96b-00c04fd705a2}</string>
    </property>
    <property name="geometry" stdset="0">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>744</width>
      <height>400</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>750</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QAxWidget</class>
   <extends>QWidget</extends>
   <header>qaxwidget.h</header>
  </customwidget>
  <customwidget>
   <class>WebAxWidget</class>
   <extends>QAxWidget</extends>
   <header>webaxwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
